site/en/reference/cli/concepts/udp.md

```diff
@@ -0,0 +1,193 @@
+# Send UDP Messages with OT CLI
+
+Thread networks offer several `udp` commands for establishing peer-to-peer communication
+using UDP sockets. Each peer needs its own socket.
+
+In addition to opening the required sockets, one requirement is that the socket
+on the receiving end of the initial communication be bound.
+
+The examples that follow demonstrate how you can open and bind sockets, how to
+to specify a peer with which to associate a local socket, and how to send messages
+using UDP sockets.
+
+## UDP Commands
+
+For a list of `udp` commands, type `help`:
+
+> udp help
+bind
+close
+connect
+linksecurity
+open
+send
+Done
+
+### `open` command
+
+Use the `udp open` command to open a socket to begin UDP communication.
```

Comment:
We should not use the term "a socket" as all `cli udp` commands interact with just one specific example socket provided by CLI.

We can refer to it as "the example socket" which make it consistent with the `README_UDP.md`.